packages/web/src/pages/saved-page/components/mobile/SavedPage.module.css

```diff
@@ -4,12 +4,20 @@
 
 .header {
   border-bottom: 1px solid var(--neutral-light-8);
+  flex-wrap: wrap;
+  height: 76px;
+}
+
+.categoryMenuWrapper {
+  flex-grow: 1;
+  overflow-x: scroll;
+  width: 100%;
 }
 
 .spinner {
   height: 40px;
   width: 40px;
-  margin: 200px auto 0;
+  margin: var(--unit-18) auto 0;
```

Comment:
Adjusted these margins to account for the addition of the filter pills in the header